Java-Assignment8/src/UserDefineException.java

```diff
@@ -0,0 +1,30 @@
+// Define three custom exceptions
+class CustomException1 extends Exception {}
+class CustomException2 extends Exception {}
+class CustomException3 extends Exception {}
+
```

Comment:
can give some valid code what cause to throw this exception
like classic exception example AgeVerification exception (> 18 or not)